README.md

```diff
@@ -60,22 +60,24 @@ in sync with the written specification on conventionalcommits.org.
 /* Any non-newline whitespace: */
 <whitespace>    ::= <ZWNBSP> | <TAB> | <VT> | <FF> | <SP> | <NBSP> | <USP>
 
-<message>       ::= <summary>, <newline>*, <body>
-                 |  <summary>
+<message>       ::= <summary>, <newline>*, <body>, <newline>*, <footer>+
```

Comment:
I think the `newlines` here aren't correct with `*`. `*` means `0` or `many`, but what happens if we add `0` newlines (none):

```
feat: this is my subject Ref #1337
```

This will be parsed as the tree below, because the `text` node only stops on newlines.

```
message
└ summary
  ├ type: "feat"
  ├ separator: ":"
  ├ whitespace: " "
  └ text: "this is my subject Ref #1337"
```

Meaning, we need `1` or `many` newlines (`+`) to break the `summary.text` node and parse the content after newline as either `footer` or `body`.

Reply:
I agree, seems like it should be `+`